### PR TITLE
PMM-7 fix imports

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-7

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: Makefile.devcontainer [hunk 1]
```diff
@@ -75,6 +75,7 @@ FILES = $(shell find . -type f -name '*.go')
 
 format:                         ## Format source code.
 	gofumpt -l -w $(FILES)
+	gci write --Section Standard --Section Default --Section "Prefix(github.com/percona/pmm-managed)" .
 	goimports -local github.com/percona/pmm-managed -l -w $(FILES)
```
Comment (idoqo) [inline Makefile.devcontainer:79]: Shouldn't we run `gci` after `goimports`? Not sure but I imagined gci will only order them and it makes sense to do that after goimports have fixed missing/unused imports or something.

```suggestion
        goimports -local github.com/percona/pmm-managed -l -w $(FILES)
	gci write --Section Standard --Section Default --Section "Prefix(github.com/percona/pmm-managed)" .
```

Comment (BupycHuk) [reply Makefile.devcontainer:79]: good point

File: Makefile.devcontainer [hunk 2]
```diff
@@ -75,7 +75,8 @@ FILES = $(shell find . -type f -name '*.go')
 
 format:                         ## Format source code.
 	gofumpt -l -w $(FILES)
-	goimports -local github.com/percona/pmm-managed -l -w $(FILES)
+        goimports -local github.com/percona/pmm-managed -l -w $(FILES)
+	gci write --Section Standard --Section Default --Section "Prefix(github.com/percona/pmm-managed)" .
```
Comment (artemgavrilov) [inline Makefile.devcontainer:79]: Redundand tab space before goimports. But do we need goimports at all now?

**Review by artemgavrilov — APPROVED** [review]
Epic!

File: Makefile.devcontainer (hunk 2 above)
Comment (idoqo) [reply Makefile.devcontainer:79]: old comment so not if still valid, but looks like we do (https://github.com/daixiang0/gci/issues/1#issuecomment-668305417)

**Review by idoqo — APPROVED** [review]

File: Makefile.devcontainer (hunk 2 above)
Comment (artemgavrilov) [reply Makefile.devcontainer:79]: Aha, got it! Thanks!

**Comment by artemgavrilov** [comment]
@BupycHuk Let's also add `gci` to the list of required linters

**Pull request merged**